pegasus/tools/visualization.py

```diff
@@ -2,12 +2,15 @@
 import numpy as np
 import scipy
 import umap as umap_module
-import forceatlas2 as fa2
 import uuid
 
 from threadpoolctl import threadpool_limits
 from pegasusio import MultimodalData
 from pynndescent import NNDescent
+try:
```

Comment:
Why do we need Line 10 to 13?

Reply:
I understand your purpose now: you want to raise `PackageNotFoundError` which is not in Python's default exceptions but from `importlib_metadata`.

Is there any particular reason you want to use `PackageNotFoundError`, while either `ImportError` or its single subclass `ModuleNotFoundError` from Python's default exceptions could cover the case? I mean, if you use `ImportError` or `ModuleNotFoundError`, you don't need to write any `import` statement.